spec/serializers/activitypub/accept_follow_serializer_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ActivityPub::AcceptFollowSerializer do
+  subject { serialized_record_json(record, described_class, adapter: ActivityPub::Adapter) }
+
+  describe 'serializing an object' do
+    let(:record) { Fabricate :follow_request }
+
+    it 'returns expected attributes' do
+      expect(subject.deep_symbolize_keys)
+        .to include(
+          actor: match(record.target_account.username),
```

Comment:
What about something more specific (and less brittle, see our discussion about removing username from new AP ids), like:
```suggestion
          actor: eq(ActivityPub::TagManager.instance.uri_for(record.target_account)),
```